packages/edge-demo-site/server.js

```diff
@@ -0,0 +1,14 @@
+// Import path interpreted by the Remix compiler
+import * as build from '@remix-run/dev/server-build'
+import { createRequestHandler } from '@netlify/remix-edge-adapter'
+
+export default createRequestHandler({
+  build,
+  // process.env.NODE_ENV is provided by Remix at compile time
+  mode: process.env.NODE_ENV,
+})
+
+export const config = {
```

Comment:
This will be included in the remix-edge template, but for now, I'm just adding it for the demo site so it works on Netlify. This config object means that all paths will be cached via Edge Function caching.